test/tls/scylla.yaml

```diff
@@ -227,7 +227,7 @@ batch_size_fail_threshold_in_kb: 50
 # PasswordAuthenticator}.
 #
 # - AllowAllAuthenticator performs no checks - set it to disable authentication.
-# - PasswordAuthenticator relies on username/password pairs to authenticate
+# - PasswordAuthenticator relies on username/password pairs to authentication
```

Comment:
This one was actually legit - "relies on username/password pairs to _authenticate_ users", here it makes sense to use the verb.

Please do not perform blind search and replace like this in comments. If you do, you need to manually verify that the changes make sense.

If you need to change the name of a variable, either use refactoring tools offered by your IDE or change the name at the definition and use the compiler to guide you to places where the names should actually be changed.